guidelines/responses.md

```diff
@@ -6,12 +6,18 @@ The API responds with `Content-Type` set to `application/json` and JSON content.
 
 ## Response codes
 
-* `400 Bad Request` - Error caused by the client app, e.g. in case of malformed request or invalid identifier of a resource. In most cases, such an error signifies a bug in the client app \(consumer of the API\).
-* `401 Unauthorized` - Error caused by usage of invalid ClientToken, AccessToken, or you may not have the necessary permission to use the endpoint.
-* `403 Forbidden` - Server error that should be reported to the end user of the client app. Happens for example when the server-side validation fails or when a business-logic check is violated.
-* `408 Request Timeout` - Error caused by heavy requests that takes too long to process (in magnitude of tens of seconds). To get around this send/request data in smaller batches.
-* `429 Too Many Requests` - Error caused by too many requests sent in a given amount of time, please see [Request limits](requests.md#request-limits) for more information.
-* `500 Internal Server Error` - Unexpected error of the server. In most cases, such an error signifies a bug on our side. We are logging it and are immediately notified when such an error happens. If anything like this happens, feel free to directly contact us or raise an issue here on Github.
+* **400 Bad Request**
+  * Error caused by the client app, e.g. in case of malformed request or invalid identifier of a resource. In most cases, such an error signifies a bug in the client app \(consumer of the API\).
+* **401 Unauthorized**
+  * Error caused by usage of invalid ClientToken, AccessToken, or you may not have the necessary permission to use the endpoint.
+* **403 Forbidden**
+  * Server error that should be reported to the end user of the client app. Happens for example when the server-side validation fails or when a business-logic check is violated.
+* **408 Request Timeout**
+  * Error caused by heavy requests that takes too long to process (in magnitude of tens of seconds). To get around this send/request data in smaller batches.
+* **429 Too Many Requests**
+  * Error caused by too many requests sent in a given amount of time. Response contains `Retry-After` header indicating how long the user agent should wait before making a follow-up request. For more information, see [Request limits](requests.md#request-limits).
+* **500 Internal Server Error**
+  * Unexpected error of the server. In most cases, such an error signifies a bug on our side. We are logging it and are immediately notified when such an error happens. If anything like this happens, feel free to directly contact us or raise an issue here on Github.
```

Comment:
Not sure if the `here on github` is correct as this can be viewed on gitbook or potntially somewhere else. So maybe something like `raise and issue on our documentation repo on Github [link]` ?

Reply:
Ha, I didn't change that part, but yes it could do with some editing. Perhaps something like this:
* Unexpected error on the Mews side. This may be due to a software fault. If such a situation occurs, the error will be logged and the development team notified, however you can raise an issue through GitHub on our [documentation repository](https://github.com/MewsSystems/gitbook-connector-api).